internal/Employee.go

```diff
@@ -0,0 +1,8 @@
+package internal
+
+type Employee struct {
+	ID         string    `json:"id"`
```

Comment:
the ID of entry shouldn't present in the "request" model. The should be ID an identifier generated by business logic of the service. 

But ID filed is required in the "response" model. 
I'd propose to have 2 types of models. The first type are request models (without ID) and second type are response models (with ID field).

Reply:
If I create new models, I am allowed to change my documentation?

Reply:
Sure. The code should be in sync with a spec

Reply:
If I do this, then my base will change, I will add new maps to it?

Reply:
+